tests/test_cookies.py

```diff
@@ -123,6 +123,37 @@ def test_cookie_samesite_none_not_secure():
         c.serialize()
 
 
+def test_cookie_samesite_future():
+    # first default
+    c = cookies.Cookie()
+    with pytest.raises(ValueError) as excinfo:
+        c[b"foo"] = b"bar"
+        c[b"foo"].samesite = b"Future"
+        c.serialize()
+    assert excinfo.value.args[0] == "SameSite must be 'strict', 'lax', or 'none'"
+
+    try:
```

Comment:
We use pytest, please use the `monkeypatch` fixture to monkeypatch during this test, and use `pytest.raises()` within the test.

Write two tests, one that tests when the flag is set, and one that tests when the flag is not set, feel free to use a pytest parametrize to test both invariants with a single test.